jobs/config.json

```diff
@@ -232,14 +232,14 @@
 "pull-kubernetes-node-e2e": {
   "scenario": "kubernetes_kubelet",
   "args": [
-    "--scenario=test/e2e_node/jenkins/jenkins-pull.properties"
+    "--script=test/e2e_node/jenkins/jenkins-pull.properties"
```

Comment:
Apparently I lost the race between Ryan committing the change for me and my pushing the fix to this branch